pkg/messages/errorcodes/errorcodes.go

```diff
@@ -42,108 +43,124 @@ func (e *ErrorCode) Error() string {
+
+	// ### Conversation (Building block)
+	ConversationInvalidParms        = ErrorCode{"ERR_CONVERSATION_INVALID_PARMS", "", CategoryConversation}  // Invalid parameters for conversation component
+	ConversationInvoke              = ErrorCode{"ERR_CONVERSATION_INVOKE", "", CategoryConversation}         // Error invoking conversation
+	ConversationMissingInputs       = ErrorCode{"ERR_CONVERSATION_MISSING_INPUTS", "", CategoryConversation} // Missing inputs for conversation
+	ConversationNotFound            = ErrorCode{"ERR_CONVERSATION_NOT_FOUND", "", CategoryConversation}      // Conversation not found
+	ConversationDirectInvoke        = ErrorCode{"ERR_DIRECT_INVOKE", "", CategoryConversation}               // Error on service invocation   #TODO - fix this after jake's PR is merged
+	ConversationInvokeOutputBinding = ErrorCode{"ERR_INVOKE_OUTPUT_BINDING", "", CategoryConversation}       // Error invoking output binding
+
+	// ### Distributed Lock (Building block)
+	LockStoreNotConfigured = ErrorCode{"ERR_LOCK_STORE_NOT_CONFIGURED", "", CategoryLock} // Lock store not configured
+	LockStoreNotFound      = ErrorCode{"ERR_LOCK_STORE_NOT_FOUND", "", CategoryLock}      // Lock store not found
+	LockTry                = ErrorCode{"ERR_TRY_LOCK", "", CategoryLock}                  // Error acquiring lock
+	LockUnlock             = ErrorCode{"ERR_UNLOCK", "", CategoryLock}                    // Error releasing lock
+
+	// ### Healthz
+	HealthNotReady         = ErrorCode{"ERR_HEALTH_NOT_READY", "", CategoryHealth}          // Dapr not ready
+	HealthAppidNotMatch    = ErrorCode{"ERR_HEALTH_APPID_NOT_MATCH", "", CategoryHealth}    // Dapr  App ID does not match
+	HealthOutboundNotReady = ErrorCode{"ERR_OUTBOUND_HEALTH_NOT_READY", "", CategoryHealth} // Dapr outbound not ready
+
+	// ### Common
+	CommonAPIUnimplemented     = ErrorCode{"ERR_API_UNIMPLEMENTED", "", CategoryCommon}      // API not implemented
+	CommonAppChannelNil        = ErrorCode{"ERR_APP_CHANNEL_NIL", "", CategoryCommon}        // App channel is nil
+	CommonBadRequest           = ErrorCode{"ERR_BAD_REQUEST", "", CategoryCommon}            // Bad request
+	CommonBodyRead             = ErrorCode{"ERR_BODY_READ", "", CategoryCommon}              // Error reading request body
+	CommonInternal             = ErrorCode{"ERR_INTERNAL", "", CategoryCommon}               // Internal error
+	CommonMalformedRequest     = ErrorCode{"ERR_MALFORMED_REQUEST", "", CategoryCommon}      // Malformed request
+	CommonMalformedRequestData = ErrorCode{"ERR_MALFORMED_REQUEST_DATA", "", CategoryCommon} // Malformed request data
+	CommonMalformedResponse    = ErrorCode{"ERR_MALFORMED_RESPONSE", "", CategoryCommon}     // Malformed response
+
+	// ### Scheduler/Jobs API
+	SchedulerScheduleJob   = ErrorCode{"SCHEDULER_SCHEDULE_JOB", "SCHEDULER_SCHEDULE_JOB", CategoryJob}     // Error scheduling job
```

Comment:
I think this is what you are wanting to prefix `DAPR_` with - correct?

```suggestion
	SchedulerScheduleJob   = ErrorCode{"SCHEDULER_SCHEDULE_JOB", "DAPR_SCHEDULER_SCHEDULE_JOB", CategoryJob}     // Error scheduling job
```

Reply:
We should probably add a string check to kit to check if `DAPR_` is prefixed and if not to add it to avoid this going forward. This was missed bc the const lives in runtime atm, but all the others live in kit and are prefixed in kit. I think I assumed that prefix was added to the const when I added it to runtime

Reply:
Since scheduler isn't stable 'yet', my vote would be update this to include the dapr prefix. Elena, could you go ahead with this since you have this PR open?

Reply:
Yes, definitely! Since we're still in Alpha now is the right time to change this.